lib/ronin/recon/cli/commands/run.rb

```diff
@@ -308,7 +313,7 @@ def run(*values)
                 end
               end
             ensure
-              output_file.close if options[:output]
+              output_files&.each(&:close)
```

Comment:
I'm not a super fan of `&.`, but I'll allow it. Much prefer the explicit `foo.do_stuff if foo`.